test/defaultMetricsTest.js

```diff
@@ -71,6 +71,21 @@ describe('collectDefaultMetrics', () => {
 		});
 	});
 
+	it('should apply labels to metrics when configured', () => {
+		const labels = {
```

Comment:
can you add a `expect.assertions(5)` (or whatever the number is)?

Reply:
https://github.com/rkoval/prom-client/commit/82a7106ec03f8b97c4ffd861d163d8b18e56ba6d

Reply:
https://jestjs.io/docs/en/expect#expectassertionsnumber - it ensures that that many assertions are made, which makes sure tests don't falsely pass if some bug causes the async callback not to run.

Reply:
do you know why this function produces inconsistent results / doesn't always have the same number of assertions? i can actually reproduce the inconsistencies locally: 
![image](https://user-images.githubusercontent.com/1282943/86540769-249c0800-becd-11ea-9802-9e18e952ed44.png)
the first run has 47 assertions, but the second only has 45 (i didn't change any code between each run)

Reply:
probably something async we're not properly waiting for

Reply:
ok, what would you like to do here? again, this type of assertion isn't used anywhere else in the codebase, so i might just have to close this PR and just use our own fork if it'll be blocked without it. i unfortunately don't have time to debug the issue

Reply:
went with a slightly different approach